src/components/property-tree/property-tree.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
```

Comment:
This can be compared to deleted file with the same name in a diff checker tool to see differences